spec/features/Users/login_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe "Logging In" do
+  it "can log in with valid credentials" do
+    user = User.create(name: "Elvis", password: "test", email: 'user@email.com')
+    visit '/'
+    fill_in :email, with: user.email
+    fill_in :password, with: user.password
+
+
+    click_on "Log In"
+
+    expect(current_path).to eq('/dashboard')
+
+    expect(page).to have_content("Welcome, #{user.name}")
+    expect(page).to have_button("Log out")
+  end
+
+  it "cannot log in with bad credentials" do
+    user = User.create(name: "Elvis", password: "test", email: 'user@email.com')
+    visit '/'
+    fill_in :email, with: user.email
+    fill_in :password, with: "incorrect password"
+
+    click_on "Log In"
+
+    expect(current_path).to eq('/')
+
+    expect(page).to have_content("Sorry, your credentials are bad.")
+
+    fill_in :email, with: user.email
+
+    click_on "Log In"
+
+    expect(current_path).to eq('/')
+
+    expect(page).to have_content("Sorry, your credentials are bad.")
+
+    fill_in :email, with: "burt"
+    fill_in :password, with: user.password
+
+    click_on "Log In"
+
+    expect(current_path).to eq('/')
+
+    expect(page).to have_content("Sorry, your credentials are bad.")
+  end
+
```

Comment:
maybe we also add a test here that the user can logout and a sad path test for displaying an error message if they try to visit the dashboard page when not logged in (ie 401/not authorized to access page)